webservices/partition/base.py

```diff
@@ -316,6 +291,7 @@ def refresh_child(cls, cycle, queue_old, queue_new):
             # populating the child tables during normal partitioning.
             # Otherwise, an error is thrown due more values being specified
             # than there are columns to accept them.
+            # VM TODO - Is this still needed?
```

Comment:
Given the other changes here and setup of the master table, it may not be.  We'll have to review the flow of the data through the processing and see if it's still an issue.  It'd be nice if it weren't!